### PR TITLE
update README for final deliverable

### PR DESCRIPTION
**Pull request by BraedenKloke**
# Description
Updates README for final deliverable.

## Issues
N/A

# How to test
RTFM.

# Checklist
**I've added tests**
- [ ] Yes

- [X] No ... N/A

**I've updated the DB schema and UML diagram**
- [X] Yes
- [ ] No

### CODE REVIEW

**Comment by BraedenKloke** [comment]
@Sarvesh-Sabapathy @AliAbdollahian @ajbalara @JackieSL1 

**What level of detail should I update the DB schema to?**

Its not polished right, and I'm sure Maede with ask about it right now.

File: diagrams/models-class-diagrams-final.png
Comment (JackieSL1) [inline diagrams/models-class-diagrams-final.png:1]: Could you please give this a white background for us vampires

**Comment by JackieSL1** [comment]
> @Sarvesh-Sabapathy @AliAbdollahian @ajbalara @JackieSL1
> 
> **What level of detail should I update the DB schema to?**
> 
> Its not polished right, and I'm sure Maede with ask about it right now.

I think whatever the auto generated schema includes should be sufficient, as long as it has all of the tables, columns with data types, PK indicators, and relationships.

**Comment by JackieSL1** [comment]
Other than my comment, LGTM!

**Review by JackieSL1 — APPROVED** [review]

**Comment by BraedenKloke** [comment]
> > @Sarvesh-Sabapathy @AliAbdollahian @ajbalara @JackieSL1
> > **What level of detail should I update the DB schema to?**
> > Its not polished right, and I'm sure Maede with ask about it right now.
> 
> I think whatever the auto generated schema includes should be sufficient, as long as it has all of the tables, columns with data types, PK indicators, and relationships.

Well it has NONE of those lol. I will get to work on it.

**Pull request merged**